### PR TITLE
Readme cover image update

### PR DESCRIPTION
**Pull request by avr757**
Update readme cover images to all samples.

- Unified cover image sizes to 1280x720 res
- Changed all image format to .png
- Changed readme files

### CODE REVIEW

File: sample_project/Content/SampleViewer/Samples/Measure/readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Display the measured distance by drawing lines on the map.
```
Comment (ZackAllen) [inline sample_project/Content/SampleViewer/Samples/Measure/readme.md:3]: Metadata.json is not changed for measure, was it already correct?

**Review by ZackAllen — CHANGES_REQUESTED** [review]

**Comment by Jade-JadeH** [comment]
Routing sample one I think the angle could be more top-down like the Unity one. 

Third person one---love it!

File: sample_project/Content/SampleViewer/Samples/Measure/readme.md (hunk 1 above)
Comment (avr757) [reply sample_project/Content/SampleViewer/Samples/Measure/readme.md:3]: Yes, the image name is already matching 
image

**Comment by avr757** [comment]
> Routing sample one I think the angle could be more top-down like the Unity one.

How about this one?


Routing

**Comment by Jade-JadeH** [comment]
I think the Unity one's angle looks better, I think it's better to not have a birdview so that it looks completely 2D.

**Comment by avr757** [comment]
> I think the Unity one's angle looks better, I think it's better to not have a birdview so that it looks completely 2D.

How about this angle?

Routing

**Comment by Jade-JadeH** [comment]
Thanks for keep working on it.

**Review by Jade-JadeH — APPROVED** [review]

**Review by ZackAllen — APPROVED** [review]

**Pull request merged**